ext/AvizExt/viz/coral_dynamics.jl

```diff
@@ -0,0 +1,275 @@
+    axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}()
+)::Figure
+    if !haskey(rs.outcomes, :relative_taxa_cover)
+        throw(ArgumentError("Unable to found relative_taxa_cover in outcomes. This variable may be passed manually."))
+    end
+    return ADRIA.viz.taxonomy(
+        rs.inputs,
+        rs.outcomes[:relative_taxa_cover];
+        opts=opts,
+        fig_opts=fig_opts,
+        axis_opts=axis_opts,
+        series_opts=series_opts
+    )
+end
+function ADRIA.viz.taxonomy(
+    scenarios::DataFrame,
+    relative_taxa_cover::YAXArray;
+    opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    fig_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}()
+)::Figure
+    fig_opts[:size] = get(fig_opts, :size, (1200, 1200))
+    f = Figure(; fig_opts...)
+
+    g = f[1, 1] = GridLayout()
+
+    _scenarios = copy(scenarios[1:end .∈ [relative_taxa_cover.scenarios], :])
+    scen_groups = if get(opts, :by_RCP, false)
+        ADRIA.analysis.scenario_rcps(_scenarios)
+    else
+        ADRIA.analysis.scenario_types(_scenarios)
+    end
+
+    ADRIA.viz.taxonomy!(
+        g,
+        relative_taxa_cover,
+        scen_groups,
+        opts=opts,
+        axis_opts=axis_opts,
+        series_opts=series_opts
+    )
+
+    return f
+end
+function ADRIA.viz.taxonomy!(
+    g::Union{GridLayout,GridPosition},
+    relative_taxa_cover::YAXArray,
+    scen_groups::Dict{Symbol, BitVector};
+    opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}()
+)::Union{GridLayout,GridPosition}
+    axis_opts[:xlabel] = get(axis_opts, :xlabel, "Year")
+    axis_opts[:ylabel] = get(axis_opts, :ylabel, "Relative Cover")
+
+    show_confints::Bool = get(opts, :show_confints, true)
+    compare_taxa::Bool = get(opts, :compare_taxa, true)
+    if compare_taxa
+        # Create colors
+        n_groups::Int64 = length(relative_taxa_cover.taxa)
+        default_color = Symbol("Set1_" * string(n_groups))
+        color = get(opts, :colors, default_color)
+        _colors = categorical_colors(color, n_groups)
+
+        # Plot results
+        taxonomy_series_confint!(
+            g,
+            relative_taxa_cover,
+            scen_groups,
+            _colors;
+            show_confints=show_confints,
+            axis_opts,
+            series_opts
+        )
+    else
+        # Use default ADRIA colors for scenario type if use not specified
+        n_scen_types::Int64 = length(keys(scen_groups))
+        color = get(opts, :colors, nothing)
+        _colors = isnothing(color) ? [
+            COLORS[scen_name] for scen_name in keys(scen_groups)
+        ] : categorical_colors(color, n_scen_types)
+
+        # Plot results
+        taxonomy_intervened_series_confint!(
```

Comment:
Same thing, you are not plotting only the scenarios that received intervention. Maybe rename it to `interventions_by_taxa` (so, one plot would be "taxa splitted by intervention" and the other "interventions splitted by taxa")?

I'm open to other ideas.